src/mpid/ch4/netmod/ofi/init_addrxchg.c

```diff
@@ -60,7 +60,12 @@ static int get_av_table_index(int rank, int nic, int vni)
             return rank;
         }
     } else {
+#ifdef MPIDI_OFI_VNI_USE_DOMAIN
         int num_vnis = MPIDI_OFI_global.num_vnis;
+#else
+        /* with scalable endpoint as context, all vnis share the same address. */
+        int num_vnis = 1;
```

Comment:
I think this variable should be named something else to avoid confusion with the actual number vnis that we use for SEP, which can still be higher than one. Maybe `num_domains`?

Reply:
This is a local variable with the scope of a few lines. And it also has a comment explaining why it is set to `1` in the case of SEP. I think it is fine. `num_domains` would be a new name that isn't used elsewhere and I think it is not necessarily easier to read.

Reply:
Hmm.. with the comment, it's probably fine.

Reply:
Its odd to look at, but I'm not sure `num_domains` or `num_eps` is that much better. I guess I agree it's probably fine as-is.